server/src/main/scala/com/metamx/tranquility/server/http/ServerMain.scala

```diff
@@ -186,9 +119,9 @@ object ServerMain extends App with Logging
 
   def createJettyServer(
     lifecycle: Lifecycle,
-    globalConfig: GlobalConfig,
+    globalConfig: ServerConfig,
     servlet: TranquilityServlet
-  ): Server =
+    ): Server =
```

Comment:
I'm not sure why your formatter is adding these two extra spaces on all the multiline method arg lists. Does applying the codestyle remove the spaces? (cmd-alt-L).

If not I'm ok letting it slide :P